assignments/lambda-classes.js

```diff
@@ -1 +1,167 @@
 // CODE here for your Lambda Classes
+
+class Person {
+  constructor(personAttributes) {
+    this.name = personAttributes.name;
+    this.age = personAttributes.age;
+    this.location = personAttributes.location;
+    this.gender = personAttributes.gender;
+  }
+  // Method
+  speak() {
+    return `Hello my name is ${this.name}, I am from ${this.location}.`;
+  }
+}// Person
+
+
+// Instructor
+class Instructor extends Person {
+  constructor(instructorAttributes) {
+    super(instructorAttributes);
+    this.specialty = instructorAttributes.specialty;
+    this.favLanguage = instructorAttributes.favLanguage;
+   this.catchPhrase = instructorAttributes.catchPhrase;
+  }
+  // Methods
+  demo(subject) {
+    return `Today we are learning about ${subject}.`;
+  }
+  grade(student, subject) {
+    return `${student.name} receives a perfect score on ${subject}`;
+  }
+
+}
+
+// Student
+class Student extends Person {
+  constructor(studentAttributes) {
+    super(studentAttributes);
+    this.previousBackground = studentAttributes.previousBackground;
+    this.className = studentAttributes.className;
+    this.favSubjects = studentAttributes.favSubjects;
+  }
+  // Methods
+  listsSubjects() {
+    return `${this.favSubjects}`; //?
```

Comment:
This works just fine because you're logging the result of the method, but I think we are looking for `.forEach(x => console.log(x))` sort of thing